source/circleci-demo/.gitignore

```diff
@@ -0,0 +1,244 @@
+
+# Created by https://www.gitignore.io/api/osx,linux,python,windows,pycharm,visualstudiocode
```

Comment:
all the `.gitignore`s in this PR are the same --- could we just use one at the root and not have to worry about them in each directory?